packages/cli/src/commands/upgrade.js

```diff
@@ -246,5 +246,6 @@ export const handler = async ({ d, tag, pr }) => {
     await tasks.run()
   } catch (e) {
     console.log(c.error(e.message))
```

Comment:
Use stderr, instead of stdout.
```suggestion
    console.error(c.error(e.message))
```